fleet-tenancy/terraform/1-fleet/variables.tf

```diff
@@ -14,8 +14,24 @@
  * limitations under the License.
  */
 
+# [START config_sync_fleet_variables]
 variable "project" {
   type = string
   description = "Name of GCP project to host the Fleet"
 }
 
+variable "gcp_sa_id" {
```

Comment:
These variables are removed by https://github.com/GoogleCloudPlatform/anthos-config-management-samples/pull/104. You need to rebase your commit.